webdriverio/web/webdriver/test/configs/wdio.saucelabs.android.conf.js

```diff
@@ -0,0 +1,67 @@
+const {config} = require('./wdio.shared.conf');
+const build = `Android Chrome WebdriverIO-V6 build-${new Date().getTime()}`
+
+// =========================
```

Comment:
I would suggest creating a `wdio.shared.sauce.conf.js`-file like [here](https://github.com/saucelabs-training/demo-js/blob/master/biometric-login/test/configs/wdio.shared.sauce.conf.js) because we now have the same code in 3 different files

Reply:
worked like a charm